### PR TITLE
Prevent files larger than 600kb from being uploaded

### LINKED ISSUE #696: Enforce size limit for subjects
I just got an email from a science trying to upload 6MB images. I think we should decide on a size limit for subjects uploaded through the project builder (and add help text with "suggested size" and "maximum size"). Talking to @camallen and @mrniaboc it sounds like 400kb should be a decent max size for standard projects, though curious what @vrooje @mschwamb are uploading currently.

### PR DESCRIPTION
**Pull request by edpaget**
Closes #696

### CODE REVIEW

File: app/pages/lab/subject-set.cjsx [hunk 1]
```diff
@@ -154,8 +156,8 @@ EditSubjectSetPage = React.createClass
           <strong>Manifest files are required.</strong><br />
           <strong>Drag-and-drop or click to upload manifests and subject images here.</strong><br />
           Manifests must be <code>.csv</code> or <code>.tsv</code>. The first row should define metadata headers. All other rows should include at least one reference to an image filename in the same directory as the manifest.<br />
-          Subject images can be any of: {<span key={ext}><code>{ext}</code>{', ' if VALID_SUBJECT_EXTENSIONS[i + 1]?}</span> for ext, i in VALID_SUBJECT_EXTENSIONS}{' '}
-          and may not contain {<span key={char}><kbd>{char}</kbd>{', ' if INVALID_FILENAME_CHARS[i + 1]?}</span> for char, i in INVALID_FILENAME_CHARS}.<br />
+          Subject images can be {MAX_FILE_SIZE/1000}kb and any of: {<span key={ext}><code>{ext}</code>{', ' if VALID_SUBJECT_EXTENSIONS[i + 1]?}</span> for ext, i in VALID_SUBJECT_EXTENSIONS}{' '}
```
Comment (brian-c) [inline app/pages/lab/subject-set.cjsx]: Pedantic: lowercase "kb" is kilobits. Can we make this "KB"? And maybe "can be up to _n_ KB" is clearer?

File: app/partials/subject-uploader.cjsx [hunk 2]
```diff
@@ -54,6 +54,7 @@ module.exports = React.createClass
 
     if subjectData? and @state.inProgress
       locationTypes = for filename in subjectData.locations
+        console.log(@props.files[filename])
```
Comment (brian-c) [inline app/partials/subject-uploader.cjsx]: Wanna lose this while you're in there?

**Comment by edpaget** [comment]
Thanks @brian-c :+1:

**Comment by brian-c** [comment]
Grand, thanks.

**Pull request merged**